### PR TITLE
Removed notifications bell

### PR DESCRIPTION
**Pull request by gusartus**
Just because we don't have notifications yet

### CODE REVIEW

**Review by CodeAnnihilator — COMMENTED** [review]
you haven't removed styles related to this code

**Comment by gusartus** [comment]
How about now?

File: src/common/components/Header/Header.tsx [hunk 1]
```diff
@@ -9,7 +9,6 @@ import i18n from 'locales/i18nextConfig';
 
 import AngleArrowDownSVG from 'assets/images/angle-arrow-down.svg';
 import MenuSVG from 'assets/images/menu.svg';
-import NotificationBellSVG from 'assets/images/notification-bell.svg';
```
Comment (Gerasimov94) [inline src/common/components/Header/Header.tsx:12]: how about delete a .svg file from assets

Comment (gusartus) [reply src/common/components/Header/Header.tsx:12]: Removed

**Review by Gerasimov94 — APPROVED** [review]

**Pull request merged**